### ISSUE TITLE
Add a setting to change the decimal separator

### ISSUE THREAD

**Issue by Wjxfi** [Darkempire78/OpenCalc#175]
![318944.jpg](https://user-images.githubusercontent.com/113500363/220197289-1a9e366c-edd0-439b-81ee-34c971626912.jpg)

by all rules there should be a period. The comma only works for words

It annoys me what's wrong here.

**Comment by Domefemia** [comment]
There is not a wrong or a right here, in English the "." is used as a decimal mark and the "," for thousands and so on, but in other languages like for example Italian and French the decimal mark is a ",".
As you can read here: https://math.stackexchange.com/questions/1566106/is-there-any-difference-in-mathematical-notations-between-french-and-english

French don't even use "." for thousands but they use a space, instead of the "." we use in Italian.
We can't please everyone here because they all have different notation

**Comment by Darkempire78** [comment]
What is your language ? The caltulator change the separator depending on the language of your phone

**Comment by throwaway-d** [comment]
> We can't please everyone here because they all have different notation

Yeah, there should be a setting for the user to choose between `,` and `.`!

> The caltulator change the separator depending on the language of your phone

Looks like the setting is already available.

**Comment by Darkempire78** [comment]
but that's because it is in English (the language of programming languages). The calculator adapts to the country

**Comment by inson1** [comment]
I would love this feature!

**Comment by flori-s** [comment]
Hi, is it okay if I start working on this issue?

**Comment by Darkempire78** [comment]
@flori-s No not at all, it would be a pleasure

**Comment by giankarlo-o** [comment]
Has this feature been added or could I work on it?

**Comment by flori-s** [comment]
Never got around to it, you can build the feature!

**Comment by clementwzk** [comment]
@giankarlo-o no, you can add it 😃
